practice-one/src/layouts/Sidebar/Sidebar.jsx

```diff
@@ -9,12 +9,12 @@ import {
   logoutIcon,
   accountAvatar,
 } from "@assets/images";
-import { TAG_LINKS } from "@data/tagLinks";
+import { TAG_LINKS } from "@data";
 import { BUTTON_VARIANTS } from "@constants/buttons";
 
 const Sidebar = () => {
   return (
-    <div className="left-sidebar">
+    <aside className="left-sidebar">
       <div className="sidebar__brand">
         <a href="/" className="brand__link">
           <img src={logoBrand} alt="Wons" className="brand__img" />
```

Comment:
do you know the SEO for html page?

Reply:
About SEO (Search Engine Optimization), I knew it can help for optimizing code and content HTML of website to improve appearance when searching on Google.

Reply:
why do n't use it for logo?

Reply:
Sorry, I don't know how to use SEO for logo.
Can you explain it for me?

Reply:
the page just one the <h1> for page and person will use for the logo render. because the logo only use one time on page.

Reply:
thank you so much, I understood